recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
-{% set version = "1.14.4" %}
-{% set maj_min_ver = ".".join(version.split(".")[:2]) %}
-{% set build = 5 %}
+{% set version = "1.14.6" %}
+{% set maj_min_patch_ver = "_".join(version.split(".")) %}
```

Comment:
I think it would be easier to just use `{{ version|replace(".", "_") }}` in the url, rather than introduce another variable.

Reply:
sure.

Reply:
```suggestion
```

Reply:
but you also need `1_14` and not just `1_14_6` this kind of churn isn't necessary.

Lets focus on the cmake imporvements and the version update.